From 6eb33ebe600b7d343c5778447753be97b9b1269d Mon Sep 17 00:00:00 2001
From: Shawn Lin <shawn.lin@rock-chips.com>
Date: Tue, 24 Nov 2020 10:11:36 +0800
Subject: [PATCH] mmc: sdhci-of-dwcmshc: Disable cmd conflict check

We don't need to check cmd conflict, so remove it.

Change-Id: Ied0619820395b5354b2fe892d7986217426a3e82
Signed-off-by: Shawn Lin <shawn.lin@rock-chips.com>
---
 drivers/mmc/host/sdhci-of-dwcmshc.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mmc/host/sdhci-of-dwcmshc.c b/drivers/mmc/host/sdhci-of-dwcmshc.c
index afbf39aa6d87..834bfc142488 100644
--- a/drivers/mmc/host/sdhci-of-dwcmshc.c
+++ b/drivers/mmc/host/sdhci-of-dwcmshc.c
@@ -242,6 +242,8 @@ static int rockchip_pltf_init(struct sdhci_host *host, struct dwcmshc_priv *priv
 				 &priv->txclk_tapnum))
 		priv->txclk_tapnum = DLL_TXCLK_TAPNUM_DEFAULT;
 
+	/* Disable cmd conflict check */
+	sdhci_writel(host, 0x0, DWCMSHC_HOST_CTRL3);
 	return 0;
 }
 
-- 
2.35.3

